api/interfaces/interfaces.go

```diff
@@ -78,6 +80,8 @@ type User struct {
 	Items []interface{} `json:"items"     bson:"items"`
 
 	Loomies                         []primitive.ObjectID `json:"loomies"   bson:"loomies"`
+	ValidationCode                  string               `json:"validationCode"  bson:"validationCode"`
+	TimeExpiration                  time.Time            `json:"timeExpiration"   bson:"timeExpiration"`
```

Comment:
I think it's better to store the expiration time as a UNIX Timestamp. You can obtain the current timestamp using the following methods:

```go
currentTimestamp := time.Now().Unix()
```

And you can parse it to time.Time using:

```go
currentTime := time.Unix(currentTimestamp, 0)
```